typings/glamorous.test.tsx

```diff
@@ -0,0 +1,21 @@
+import glamorous from "../";
+
```

Comment:
Sorry I think I misunderstood your question. So this file is used to validate the type definitions by running `npm run test:ts`. This command will fail if type definitions are not correct.